lib/transaction/api_call.rb

```diff
@@ -25,33 +25,4 @@ def transactions(account_id = nil, since = nil, count = 1000, offset = 0, includ
   def get_transaction(account_id, transaction_id)
```

Comment:
this endpoint can be called either with or without account_id.
+ you are missing the `cents` filter (do not forget to update the doc)

Reply:
same for `transactions` endpoints, params have changed.
(similar to: 
https://github.com/mikamai/node-figo/blob/bc1a71c38d7e9d7cd08de74383a9da09fc1a0921/lib/figo.js#L979)